testsuite/smoke-test/src/aptos_cli/governance.rs

```diff
@@ -0,0 +1,241 @@
+        operator_cli_index,
+        Some(owner_cli_index),
+        operator_keys.consensus_public_key(),
+        operator_keys.consensus_proof_of_possession(),
+    )
+    .await
+    .unwrap();
+
+    cli.update_validator_network_addresses(
+        operator_cli_index,
+        Some(owner_cli_index),
+        HostAndPort {
+            host: DnsName::try_from("0.0.0.0".to_string()).unwrap(),
+            port,
+        },
+        operator_keys.network_public_key(),
+    )
+    .await
+    .unwrap();
+
+    cli.join_validator_set(operator_cli_index, Some(owner_cli_index))
+        .await
+        .unwrap();
+
+    reconfig(
+        &rest_client,
+        &transaction_factory,
+        swarm.chain_info().root_account(),
+    )
+    .await;
+
+    // Init package should not fail, otherwise panic.
+    cli.init_move_dir();
+    cli.init_proposal_package(PACKAGE_NAME, SCRIPT_NAME).await;
+    let hash = match cli.prepare_proposal().await {
+        Ok(ScriptHash { hash, bytecode }) => {
+            assert!(!hash.is_empty());
+            assert!(!bytecode.is_empty());
+            hash
+        }
+        Err(err) => panic!("Error preparing proposal: {:?}", err),
+    };
+
+    // Now we are ready to propose
+    let transaction = cli
+        .submit_proposal(
+            voter_cli_index,
+            cli.account_id(owner_cli_index).to_hex_literal(),
+            hash.clone(),
+        )
+        .await
+        .unwrap();
+
+    let events = match transaction {
+        Transaction::UserTransaction(ref txn) => txn
+            .events
+            .iter()
+            .filter(|event| {
+                event.typ
+                    == MoveType::Struct(MoveStructTag::new(
+                        Address::from_str("0x1").unwrap(),
+                        IdentifierWrapper::from_str("aptos_governance").unwrap(),
+                        IdentifierWrapper::from_str("CreateProposalEvent").unwrap(),
+                        vec![],
+                    ))
+            })
+            .collect::<Vec<_>>(),
+        _ => panic!("Encountered unexpected transaction type"),
+    };
+
+    assert_eq!(events.len(), 1);
+
+    let event = *events.get(0).unwrap();
+
+    let proposal_id = event.data["proposal_id"]
+        .as_str()
+        .unwrap()
+        .parse::<u64>()
+        .unwrap();
+
+    // Now lets vote
+    let transaction = cli
+        .submit_vote(
+            voter_cli_index,
+            cli.account_id(owner_cli_index).to_hex_literal(),
+            proposal_id,
+        )
+        .await
+        .unwrap();
+
+    match transaction {
+        Transaction::UserTransaction(ref txn) => assert!(txn.info.success),
+        _ => panic!("Encountered unexpected transaction type"),
+    };
+
+    assert_eq!(min_stake(&rest_client).await, MIN_STAKE);
+
+    let transaction = cli
+        .execute_proposal(
+            voter_cli_index,
+            format!(
+                "{}/build/{}/bytecode_scripts/{}.mv",
```

Comment:
Technically you should build up the pathbuf and display that so it becomes platform agnostic on path, but I bet these assumptions are taken everywhere